index.rst

```diff
@@ -1,57 +1,435 @@
+The Commissioning use case, because of its complexity and significance is described in its `own section <commissioning-analytics>`_.
+The `following sections <summit-resources>` give more details about the services and resources available to execute the use cases.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
 
-   Feel free to delete this instructional comment.
+.. _satisfying-use-cases:
 
-:tocdepth: 1
+Satisfying the Use Cases
+========================
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+.. _satisfying-active-optics:
+
+Active Optics
+-------------
+
+For more details, see the `Active Optics Data Flows <https://confluence.lsstcorp.org/x/SQfKBg>`_ Confluence page.
+
+Wavefront sensor images from LSSTCam for the AOS, during normal science operations, are delivered by a copy of the `Camera Control System's (CCS) <ccs>`_ `Data Acquisition (DAQ) Image Driver <ccs-daq-driver>`_ resident on an `AOS machine <tssw-machines>`_ at the Summit.
+The driver triggers ingestion into a `Butler repository <butler-repo>`_ local to that machine and also sends an "image available" event via `SAL <sal>`_ that includes the necessary information to form a Butler DataId.
+The AOS processing code retrieves the image via the Data Butler.
+Any master calibrations needed for image signature removal (ISR) are copied by a separate process from the `Data Backbone (DBB) <dbb>`_ and ingested into the Butler repository.
+
+For full-frame processing with ComCam and LSSTCam, an AOS script is expected to control the entire system.
+It sends "take images" commands via the `ScriptQueue CSC <script-queue>`_ to the CCS.
+the science pixels travel via the `Archiver CSC <archiver>`_ to the `Observatory Operations Data Service (OODS) <oods>`_, both at the Base.
+The Archiver sends an "image available" event via SAL that includes the necessary information to form a Butler DataId; this information is also available from the CCS via SAL messages.
+The AOS then uses the `OCS-Controlled Batch <ocs-batch>`_ service to process the data.
+The batch-executed pipelines use the Data Butler to retrieve images from the OODS.
+Any master calibrations needed are also obtained from the OODS.
+
+
+.. _satisfying-guider-centroiding:
+
+Guider Centroiding
+------------------
+
+Guider region-of-interest (ROI) images are obtained via a custom Camera DAQ client using a special guider interface.
+The image pixels in memory are converted to an `lsst.afw.Image` or `lsst.afw.Exposure` and passed to centroiding routines in the LSST Science Pipelines Libraries.
+If ISR is required, the master calibrations are pre-loaded from a local Butler repository.
+
+.. _satisfying-auxtel-control:
+
+Auxiliary Telescope Control
+---------------------------
+
+There are a few alternatives here.
```

Comment:
What's the intended audience and use of this document?

Are you soliciting input on which of these alternatives to implement? If so, who is providing that input, and who is making this decision? Is making the decision part of completing this document, or will this document be “complete” with this unresolved?

Is this an implementation-time decision (“any of these would be fine, leave it up to the developer”)? If so, let's say so.

If at all possible, it seems like it would be a good idea to take this opportunity to choose one of these alternatives, rather than punting it into the future.

Reply:
The idea behind the current formulation of the document was to describe the system that is being delivered.  In cases where that system may be inadequate for capability, performance, or schedule reasons, I made an attempt to propose technically-reasonable and resource-feasible alternatives that could be requested via appropriate change control processes.  I would expect the primary drivers of such a request to be the end users.